src/controller/cve.controller/cve.controller.js

```diff
@@ -344,8 +349,8 @@ async function updateCna (req, res, next) {
     if (cveRecord.cveMetadata.state !== CONSTANTS.CVE_STATES.PUBLISHED) {
       cveRecord.cveMetadata.state = CONSTANTS.CVE_STATES.PUBLISHED
     }
-    const providerMetadata = Object.assign({}, cveRecord.containers.cna.providerMetadata)
-    providerMetadata.dateUpdated = (new Date()).toISOString()
+
+    const providerMetadata = createProviderMetadata(orgUuid, req.ctx.org, (new Date()).toISOString())
```

Comment:
Similarly, please create a timestamp var and use it for all timestamps in updateCna